actions/executeInstructions.ts

```diff
@@ -0,0 +1,54 @@
+import { Proposal, ProposalTransaction } from '@solana/spl-governance'
+
+import { withExecuteTransaction } from '@solana/spl-governance'
+import { RpcContext } from '@solana/spl-governance'
+import { ProgramAccount } from '@solana/spl-governance'
```

Comment:
regroup the 5 `@solana/spl-governance` imports

Reply:
hmm I should find a way to configure the IDE to do it by itself, looks like a loss of time to do it manually (fix the IDE auto import)